source/common/config/utility.cc

```diff
@@ -256,6 +256,16 @@ envoy::api::v2::ClusterLoadAssignment Utility::translateClusterHosts(
   return load_assignment;
 }
 
+namespace {
+absl::string_view ProtoTypeUrlToDescriptorFullName(absl::string_view type_url) {
```

Comment:
style nit: lowerCamelCase for function names

Reply:
done